src/python/grpcio/tests/_runner.py

```diff
@@ -105,6 +105,8 @@ def write_bypass(self, value):
     Arguments:
       value (str): What to write to the original file.
     """
+    if sys.version_info.major >= 3 and not isinstance(value, bytes):
```

Comment:
On closer inspection I think I may be misremembering this, confusing it with a different place where I had to coerce bytearrays into bytes -- I think this function is getting `str` values on python3. I'll switch to using `six.binary_type`.